rpm/argparse-manpage.spec.tpl

```diff
@@ -85,8 +89,20 @@ Summary:        %{sum Python 3}
 %{desc}
 
 
+%if %{with pyproject}
+%pyproject_extras_subpkg -n python3-%{name} setuptools
+%endif
+
+
 %prep
 %setup -q
+%if %{without pyproject}
+sed '
+    /.*extras_require={/,/},$/s/.*/DROP/                  # replace all extras_require= lines with DROP
+    1,/DROP/s/DROP/    install_requires=["setuptools"],/  # replace the first DROP line with the hard dep
+    /DROP/d                                               # drop the rest lines
+' -i setup.py
+%endif
```

Comment:
Why do you do this?

Reply:
Hmm, python was complaining on EL7 it doesn't recognize `extras_require`, but I fail to reproduce now!